### ISSUE TITLE
Timestamp of message as title attribute to chat messages no longer working post v0.1.0

### ISSUE THREAD

**Issue by ghost** [owncast/owncast#3848]
### Share your bug report, feature request, or comment.

Hovering over an a message in chat no longer provides a timestamp as intended per #213. This would be a great quality of life feature if a stream is live for an extended period of time and you (or another chat user) are unsure of when a message is sent.

**Comment by cnfairall** [comment]
hi! I would like to work on this.

**Comment by gabek** [comment]
Fantastic! Thanks so much, @cnfairall! Let me know if you have any questions.

**Comment by cnfairall** [comment]
hey, I don't think I'll be able to do this ticket. I tried for over a week to set up my environment but it proved too complicated for my knowledge level. sorry about it!

**Comment by Aelly-A** [comment]
Hey, could I work on this?

**Comment by gabek** [comment]
Absolutely, thanks @Aelly-A!

**Comment by Aelly-A** [comment]
I've got a fix working, I couldn't figure out why the title wasn't appearing within the Tooltip so I just pulled it up into the div that contains the tooltip. I don't know the exact functionality before but now hovering over the text or just the colored rect that the message is in will pop up the formatted time stamp. 

The only downside of doing it this way is that hovering over the username now has both the timestamp for the message and when the user joined appear, as shown in the photo below.
<img width="325" alt="Screenshot 2024-09-17 at 2 15 30 PM" src="https://github.com/user-attachments/assets/58dd86b5-9daf-4fbe-b0bf-f3d15bbbf739">

Are we happy with this functionality for a PR or should I keep working on this?
